layouts/partials/nav_toggle.html

```diff
@@ -22,8 +22,21 @@
   {{ end }}
   <div class="wrapper-logo-mobile">
     {{- with .Page.Params.logo | default .Site.Params.logo | default "https://www.eclipse.org/eclipse.org-common/themes/solstice/public/images/logo/eclipse-foundation-white-orange.svg" }}
-      <a class="navbar-brand visible-xs" title="{{ $.Site.Title }}" href="{{ "" | absLangURL }}">
-        <img width="{{ $.Page.Params.logo_width | default $.Site.Params.logo_width | default "140"}}" class="logo-eclipse-default-mobile img-responsive" src="{{ . | absURL }}" alt="{{ $.Site.Title }}" />
+      {{ $currentPageLogo := . }}
+      {{ $currentPageLogoTitle := $.Site.Title }}
+      {{ $currentPageLogoLink := "" | absLangURL }}
```

Comment:
I wouldn't double URL convert this (you call absLangURL later on as well). Might do something unexpected, remove it here and just leave it in the later line